libs/ui-toolkit/src/components/lozenge/transaction-details.tsx

```diff
@@ -0,0 +1,22 @@
+import { formatNumber, toBigNum } from '@vegaprotocol/react-helpers';
+import type { Asset } from '@vegaprotocol/types';
+
+type TransactionDetailsProps = {
+  label: string;
+  amount: string;
+  asset: Pick<Asset, 'symbol' | 'decimals'>;
+};
+export const TransactionDetails = ({
```

Comment:
suggestion (non blocking): why not just use lozenge here rather than introducing a specific new lozenge component for transactions?

where you use <TransactionDetails/> you could probably easily do something like the below and avoid having to maintain a similar lozenge type component.

```
const label = transactionLabels[VegaTransactionType.WITHDRAW];
const num = transactionDetails.withdrawSubmission.amount;
return (
        <div className="mt-1">
          <Lozenge>
            {label} {num} {asset.symbol}
          </Lozenge>
        </div>
      );
```